src/page/Home.jsx

```diff
@@ -0,0 +1,36 @@
+import { useNavigate } from "react-router";
+import "../style/banner.css";
+import "../style/flex.css";
+
+export const Home = () => {
+    const navigate = useNavigate();
+    const navigateToLogin = () => navigate("/login");
+    const navigateToRegister = () => navigate("/register");
```

Comment:
Since you have already extracted the navigate function from useNavigate(), I feel like line 7 and 8 are not necessary.